main.tf

```diff
@@ -17,6 +17,9 @@ resource "aws_elasticache_replication_group" "redis" {
   maintenance_window            = "${var.redis_maintenance_window}"
   snapshot_window               = "${var.redis_snapshot_window}"
   snapshot_retention_limit      = "${var.redis_snapshot_retention_limit}"
+  tags                          = "${merge(var.tags, map("Name", format("tf-elasticache-%s-%s",
+                                                                        var.name,
+                                                                        lookup(data.aws_vpc.vpc.tags,"Name",""))))}"
```

Comment:
The order of tags should be different. It should be always possible to override standard tags provided by this module, for example (from another module):
```
tags = "${merge(map("Name", format("%s-%s", var.name, var.private_route_table_suffix)), var.tags, var.private_route_table_tags)}"
```